test/flows/check_building_safety_costs_flow_test.rb

```diff
@@ -12,54 +12,54 @@ class CheckBuildingSafetyCostsFlowTest < ActiveSupport::TestCase
     assert_rendered_start_page
   end
 
-  context "question: developer_agreed_to_pay?" do
-    setup { testing_node :developer_agreed_to_pay? }
+  context "question: building_over_11_metres?" do
+    setup { testing_node :building_over_11_metres? }
 
     should "render the question" do
       assert_rendered_question
     end
 
     context "next_node" do
-      should "have an outcome of developers_pay? if yes" do
-        assert_next_node :developers_pay, for_response: "yes"
+      should "have a next node of developers_agreed_to_pay? if yes" do
+        assert_next_node :developer_agreed_to_pay?, for_response: "yes"
       end
 
-      should "have an outcome of building_over_11_metres if no" do
-        assert_next_node :building_over_11_metres?, for_response: "no"
-      end
-
-      should "have an outcome of building_over_11_metres if dont_know" do
-        assert_next_node :building_over_11_metres?, for_response: "dont_know"
+      should "have an outcome of unlikely_to_need_to_pay if no" do
+        assert_next_node :unlikely_to_need_to_pay, for_response: "no"
       end
     end
   end
 
-  context "question: building_over_11_metres?" do
+  context "question: developer_agreed_to_pay?" do
     setup do
-      testing_node :building_over_11_metres?
-      add_responses developer_agreed_to_pay?: "no"
+      testing_node :developer_agreed_to_pay?
+      add_responses building_over_11_metres?: "yes"
     end
 
     should "render the question" do
       assert_rendered_question
     end
 
     context "next_node" do
-      should "have a next node of owned_by_leaseholders? if yes" do
-        assert_next_node :owned_by_leaseholders?, for_response: "yes"
+      should "have an outcome of developers_pay? if yes" do
+        assert_next_node :developers_pay, for_response: "yes"
       end
 
-      should "have an outcome of unlikely_to_need_to_pay if no" do
-        assert_next_node :unlikely_to_need_to_pay, for_response: "no"
+      should "have an outcome of owned_by_leaseholders if no" do
```

Comment:
this should be a "next node" rather than an outcome, but no big deal!

Reply:
Ah, good spot! I've updated this and the test below which were incorrectly referencing outcome